### DISCUSSION TITLE
How to upgrade with proxy

### DISCUSSION THREAD

**Discussion by lisiur** [tauri-apps/tauri#6563] [Q&A]
It is possible to upgrade the Tauri app through a terminal startup by configuring the proxy environment. But how can i upgrade the Tauri app using proxy when launching it through desktop shortcut?

**Comment by FabianLars** [comment]
There is no built-in way for that. There are a few related issues/feature requests tho:
- https://github.com/tauri-apps/tauri/issues/5433
- https://github.com/tauri-apps/tauri/issues/6467
- https://github.com/tauri-apps/tauri/issues/6470 - this one may be interesting, does the HTTP_PROXY env var works for you? You could set in rust before the tauri::Builder line. (you may have to add the `reqwest-client` feature flag to the tauri dependency in cargo.toml)
